sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashJoin.scala

```diff
@@ -186,6 +186,13 @@ trait HashJoin extends BaseJoinExec {
   private def antiJoin(
       streamIter: Iterator[InternalRow],
       hashedRelation: HashedRelation): Iterator[InternalRow] = {
+    // fast stop if isOriginInputEmpty = true
+    // whether isNullAwareAntiJoin is true or false
```

Comment:
Do we need to highlight NAAJ here or can the comment on line 190 be dropped ?

Reply:
done removed